neard/res/genesis_config.json

```diff
@@ -228,4 +232,4 @@
   "protocol_treasury_account": "test.near",
   "fishermen_threshold": "10000000000000000000000000",
   "records": []
-}
\ No newline at end of file
+}
```

Comment:
@bowenwang1996 what's going on with our fmt configs :)

Reply:
Btw, leaving the newline at the end of file is a good practice in general. We have it for Rust files.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline